apps/services/personal-representative/src/app/modules/right-types/e2e/right-types.spec.ts

```diff
@@ -0,0 +1,159 @@
+describe('Dummy', () => {
+  it('dummy', () => {
+    expect(1).toBe(1)
+  })
+})
+/*
```

Comment:
Enable this?

Reply:
@darrikonn There is a problem with testing api that does not have it's own db and is using auth-api db.
Looking into solutions but until then this is disabled.

Reply:
Same issue if you use the `@island.is/testing/nest` with the `useDatabase` hook and choose `sqlite` as the database type?

checkout `apps/services/auth-public-api/src/app/modules/delegations/test/integration/delegations.controller.spec.ts` for reference.

Reply:
Was using that one as a reference but was always stopped because of the AppModule is not a Type<any> error.
However, figured out to use the "diagnostics: false" flag in the ts-jest setup which allows the test to run anyway and everything should be passing now :)